Tests/DatadogTests/Matchers/RUMEventMatcher.swift

```diff
@@ -6,6 +6,7 @@
 
 import Foundation
 import XCTest
+import Datadog
```

Comment:
I think we can't let this import here. The comment below says:

> Note: this file is individually referenced by integration tests target, so no dependency on other source files

and IIRC this won't build in UI tests (the UI tests runner doesn't link `Datadog`).

It seems to be a leftover after some other changes, no? I don't see any `Datadog` symbols used below.